### PR TITLE
different worker names for celery workers

### PR DESCRIPTION
**Pull request by gzork**
Fixes https://github.com/plotly/streambed/issues/11462 

This is basically to have different names for workers in the Procfile, and change the docs to show how to scale both with their names..

@cldougl and @chriddyp I know you both have a ton of certainly higher prio reviews to do, don't worry about this, this is low prio whenever you have some time, as I'll notify the client also directly..

### CODE REVIEW

**Comment by chriddyp** [comment]
maybe we should just use the `DOKKU_SCALE` file instead of having the ssh step?

**Comment by chriddyp** [comment]
otherwise 💃

File: README.md [hunk 1]
```diff
@@ -63,11 +63,10 @@ But please only do this in development, not in a production environment.
 
 In order to have this setup when deploying your dash app to your Plotly On-Prem server, your `Procfile` has to execute these three processes (`celery worker`, `celery beat` and the dash app)
 
-In order to run Celery tasks On your Plotly On-Prem server, worker has to be running and linked to your dash app. To do this, scale
-up your app's "worker" processes:
+In order to run Celery tasks On your Plotly On-Prem server, your processes workers have to be running and linked to your dash app. To do this, scale up your app's workers processes (Using the names your have in your `Procfile`):
```
Comment (chriddyp) [inline README.md]: also this should be `Dash Deployment Server`

**Comment by gzork** [comment]
@chriddyp Thanks for the suggestion, I added the `DOKKU_SCALE` file here, and changed the documentation also accordingly

**Pull request merged**